components/formatted_markdown_message.jsx

```diff
@@ -21,7 +21,7 @@ export class CustomRenderer extends marked.Renderer {
             return text;
         }
         if (href[0] === TARGET_BLANK_URL_PREFIX) {
-            return `<a href="${href.substring(1, href.length)}" rel="noreferrer" target="_blank">${text}</a>`;
+            return `<a href="${href.substring(1, href.length)}" rel="noopener noreferrer" target="_blank">${text}</a>`;
```

Comment:
Technically, `noreferrer` implies `noopener`, but no harm in including both of them.